.github/workflows/publish.yml

```diff
@@ -0,0 +1,192 @@
+name: Publish to Cloudsmith
+
+on:
+  push:
```

Comment:
Hmm, we use the `publish` step instead for making the releases live. Should this be using `publish` instead of tag push?

I think that might make more sense. That way it's clear when you hit `publish`, you're making that new version of Firezone available. Reduces the steps that the release engineer in charge (me for now) makes a publish error or a security issue finds its way into a distributed package.

I think right now if a developer accidentally pushes a tag up, `apt` or `yum` will see that as a viable upgrade version in their package cache.